src/common/feedGenerator.ts

```diff
@@ -150,13 +153,30 @@ export const feedToFilters = async (
     )
     .map((setting) => setting.options.type);
 
+  // Split memberships by hide flag
+  const membershipsByHide = memberships.reduce(
+    (
+      acc: { true: string[]; false: string[] },
+      value: { sourceId: SourceMember['sourceId']; hide: boolean },
+    ) => {
+      acc[value.hide ? 'true' : 'false'].push(value.sourceId);
+      return acc;
+    },
+    { true: [], false: [] },
+  );
+
+  // If the user is not a member of the watercooler, hide it
+  if (!membershipsByHide.false.includes(WATERCOOLER_ID)) {
```

Comment:
This had me reading this section 4 times 🤣

Reply:
lol this is why i added the comment, show i change true/false to show/hide. will make it more explicit